aea/configurations/base.py

```diff
@@ -1204,6 +438,7 @@ class ConnectionConfig(ComponentConfiguration):
 
     default_configuration_filename = DEFAULT_CONNECTION_CONFIG_FILE
     package_type = PackageType.CONNECTION
+    schema = "connection-config_schema.json"
```

Comment:
Needs to be defined in constants and reused here

Reply:
done